CMakeLists.txt

```diff
@@ -102,11 +102,11 @@ else()
 endif()
 
 if(APPLE)
-    set(FreeOrionSDK_VERSION 13)
+    set(FreeOrionSDK_VERSION 14)
     set(FreeOrionSDK_FILE "FreeOrionSDK_${FreeOrionSDK_VERSION}_Clang-MacOSX-10.9-x86_64.tar.bz2")
     if(NOT EXISTS "${CMAKE_BINARY_DIR}/${FreeOrionSDK_FILE}")
     file(DOWNLOAD
-            "https://github.com/freeorion/freeorion-sdk/releases/download/v${FreeOrionSDK_VERSION}/${FreeOrionSDK_FILE}"
+            "https://github.com/o01eg/freeorion-sdk/releases/download/v${FreeOrionSDK_VERSION}/${FreeOrionSDK_FILE}"
```

Comment:
@o01eg , this line point to your fork

Reply:
Yes, it to test if MacOS works with upcoming SDK release. It will be changed back after the release.